src/main/java/codegym/tequila/fisioapp/controller/PatientController.java

```diff
@@ -0,0 +1,29 @@
+package codegym.tequila.fisioapp.controller;
+
+import codegym.tequila.fisioapp.model.MedicalRecord;
+import codegym.tequila.fisioapp.model.Patient;
+import codegym.tequila.fisioapp.service.PatientService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/patient")
+public class PatientController {
+
+    private final PatientService patientService;
+
+    public PatientController(PatientService patientService) {
+        this.patientService = patientService;
+    }
+
+    @GetMapping("/{Id}/medical-record")
+    public ResponseEntity<MedicalRecord> getPatientMedicalRecord(@PathVariable String Id) {
```

Comment:
Parameter Id is not following format conventions ... it should be lowerCamelCase